src/constants.js

```diff
@@ -145,6 +145,12 @@ export const RGBADepthPacking = 3201;
 export const TangentSpaceNormalMap = 0;
 export const ObjectSpaceNormalMap = 1;
 
+// Color space string identifiers, matching CSS Color Module Level 4 and WebGPU names where available.
+export const sRGBColorSpace = 'srgb';
+export const LinearSRGBColorSpace = 'srgb-linear';
+export const DisplayP3ColorSpace = 'display-p3';
+export const LinearACESCGColorSpace = 'acescg';
```

Comment:
I would name the constant `ACESCGColorSpace`, which is the proper name.

Also, so users do not have to remember when "s" is capitalized and when it is not, I would name it `SRGBColorSpace`.

Reply:
`SRGBColorSpace` would be inconsistent with `sRGBEncoding` — is that OK?

Sounds good on `ACESCGColorSpace`.

Reply:
`sRGBEncoding` should be removed.

Reply:
Remaining values in the encoding enum are:

- THREE.LinearEncoding
- THREE.sRGBEncoding
- THREE.BasicDepthPacking
- THREE.RGBADepthPacking

I'm not really familiar with the packing types... would they remain as encodings, or become something else? Setting that aside, removing `texture.encoding` we would have:

- `texture.colorSpace = SRGBColorSpace | LinearSRGBColorSpace`
- `renderer.outputColorSpace = SRGBColorSpace | LinearSRGBColorSpace`

I agree these are clearer and more correct names, but am not sure whether to bundle all of this into one three.js release or not.

Reply:
Not all textures have a color space, so you would have to allow for that with something like `THREE.RawData` or `THREE.NonColorData` -- or support a `null` value to indicate non-color data for the `.colorSpace` property.

Reply:
Great point – I think @gkjohnson had suggested that as well actually. I believe `null` would work well, or maybe `THREE.NoColorSpace`? This should probably be the default, rather than linear, in my opinion.

Reply:
> I believe `null` would work well, or maybe `THREE.NoColorSpace`

I far prefer the explicit and more understandable use of an enum name here like `THREE.NoColorSpace` or `THREE.RawData`.

Reply:
We might be down to aesthetics at this point. 🙂 I think any of these names are correct enough, and I'm happy to go with any of them. The only downside I see to `RawData` or `NonColorData` is that they don't have the same suffix as other `*ColorSpace` enums, but that's fine with me. Compare the four "in context":

```
texture.colorSpace = null;         // (a)
texture.colorSpace = NonColorData; // (b)
texture.colorSpace = RawData;      // (c)
texture.colorSpace = NoColorSpace; // (d)
```

Reply:
I'm not too keen on using "null" myself. (d) `NoColorSpace` would be my preferred option of the above.